example.groovy

```diff
@@ -1,3 +1,4 @@
-void sayHello() {
+void sayH ello() {
```

Comment:
📝 **[npm-groovy-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class should be marked with one of @GrailsCompileStatic, @CompileStatic or @CompileDynamic  CompileStatic